modules/serverless-customize-labels-annotations-routes.adoc

```diff
@@ -0,0 +1,68 @@
+[id="serverless-customize-labels-annotations-routes_{context}"]
+= Customizing labels and annotations for {product-title} routes
+
+{product-title} routes support the use of custom labels and annotations, which can be configured by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
+
+The following annotations are supported by Knative Serving:
+
+* Disabling routes, which is specified by the `serving.knative.openshift.io/disableRoute` annotation.
+* Enabling pass-through routes, which is specified by the `serving.knative.openshift.io/enablePassthrough` annotation.
```

Comment:
This section is misleading here. Previous paragraph says that any labels and annotations are passed. This paragraph list here some special annotations that have special effect, but that isn't full list (cluster local annotation is missing). Also, the description on `enablePassthrough` annotation is invalid - it should include information that this is about TLS termination passthough.

I think the best would be to remove this paragraph, or at least correct the description.

Reply:
OK. I was just adding these for the sake of completeness, when I asked Markus these were the Knative specific ones he mentioned, but I'll remove it and we can iterate on this another time if we need to.